### PR TITLE
Ember Routing Diagnostic Response

### PR DESCRIPTION
**Pull request by soynog**
Comfort: 3
Clarity: 4

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -6,21 +6,23 @@ Record your responses inside the fenced code blocks below each question.
...
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    You might add something like
+    {{#link-to 'campus'}}Back to All Campuses{{/link-to}}
+    in your template file under /campus/boston
```
Comment (laurenfazah) [inline diagnostic.md:25]: To link to /campus/boston, we'd want to use `{{#link-to 'campus.boston'}}Boston{{/link-to}}`. That's the appropriate answer if we were to link just to 'campus', though

File: diagnostic.md [hunk 2]
```diff
@@ -48,11 +50,11 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    In the movie route.js file, we can pass params to the model function and get the movie_id from the params object.
     ```
```
Comment (laurenfazah) [inline diagnostic.md:54]: Perfect. I.e., `params.movie_id`

**Comment by laurenfazah** [comment]
Great work, just missed what Q3 was asking for.  5/6

File: diagnostic.md (hunk 1 above)
Comment (soynog) [inline diagnostic.md:25]: Why would we want to link to Boston from Boston?

**Pull request closed without merging**